openlibrary/tests/core/test_db.py

```diff
@@ -516,6 +519,16 @@ def setup_method(self):
     def teardown_method(self):
         self.db.query('delete from yearly_reading_goals')
 
+    def test_total_yearly_reading_goals(self):
+        assert YearlyReadingGoals.total_yearly_reading_goals()['count(*)'] == 3
+        # Combination of issues
+        assert (
+            YearlyReadingGoals.total_yearly_reading_goals(since="2022-12-25")[
+                'count(*)'
+            ]
+            == 1
+        )
+
```

Comment:
```suggestion
```
Just remove this test.  The DB used for this test is not mocked --- we spin up an instance of sqlite for these tests, but we use Postgres in production.  It looks like the key for the counts is different depending on which DB is used.

Reply:
Ah I see, that explains why they are different in the 2 places - but if that is the case, why don't we just mock the Postgres DB? Is there a specific reason?